packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -64,6 +64,19 @@ export class DataSyncPlugin extends GraphbackPlugin {
                     lastSync: GraphQLNonNull(GraphQLString),
                     filter: filterTypeName
                 })
+
+                // Add updatedAt arg to update and delete input types
```

Comment:
Instead of inline comments function will be preferred.

Reply:
Good point, I will add in this and then restore conflict resolution and push again